src/main/java/dk/xam/jbang/Main.java

```diff
@@ -282,8 +337,10 @@ static void build(Script script, Main m) throws IOException {
 			}
 			optionList.addAll(Arrays.asList("-d", tmpJarDir.getAbsolutePath()));
 			optionList.addAll(Arrays.asList(script.backingFile.getPath()));
-
 			Util.info("Building jar...");
+			if (!checkVersions()) {
+				Util.warnMsg("java and javac versions are different!!");
```

Comment:
you should include the version info here.

Reply:
Yes i have changed the structure of the method a little bit to print out the version info too.